measurements/pages/__init__.py

```diff
@@ -275,6 +275,18 @@ def generator():
 
 @pages_blueprint.route('/files/download/<path:textname>')
 def files_download(textname):
+    if len(textname.split('/')) == 1:
+        # This is for backward compatibility with the new pipeline.
+        # See: https://github.com/TheTorProject/ooni-measurements/issues/44
+        q = current_app.db_session.query(Report.textname)\
+                .filter(Report.textname.endswith(textname))
+        first = q.first()
+        if first is None:
+            raise NotFound("No file with that filename found")
+
+        return redirect('/files/download/%s' % first[0])
+
+    textname_filter = (Report.textname == textname)
```

Comment:
Is it debugging line? I don't see if it's used variable...

Reply:
Ooops, yes should delete it.